config/game_options.txt

```diff
@@ -256,7 +256,7 @@ ALLOW_AI
 
 ## How long the delay is before the Away Mission gate opens. Default is half an hour.
 ## 600 is one minute.
-GATEWAY_DELAY 18000
+GATEWAY_DELAY 100
```

Comment:
this is for local testing so just ignore this

Reply:
no